src/components/field/svg/FieldGeneratedLines.tsx

```diff
@@ -13,45 +13,45 @@ class FieldPathLines extends Component<Props, State> {
   declare context: React.ContextType<typeof DocumentManagerContext>;
 
   render() {
+    const path = this.context.model.document.pathlist.activePath;
     let generatedPathString = "";
-    const generated = this.context.model.document.pathlist.activePath.generated;
-    this.context.model.document.pathlist.activePath.generated.forEach(
-      (point) => {
-        generatedPathString += `${point.x},${point.y} `;
-      }
-    );
+    const trajectory = path.generating
+      ? path.generationProgress
+      : path.generated;
+    // preserve the acccess of generationIterationNumber
+    // to trigger rerenders when mutating the in-progress trajectory in place
+    //eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
```suggestion

```

Reply:
why is this still here?

Reply:
Because otherwise eslint fails because we don't use this value